opennlp-dl/src/test/java/opennlp/dl/namefinder/NameFinderDLEval.java

```diff
@@ -54,7 +54,7 @@ public void tokenNameFinder1Test() throws Exception {
     Assert.assertEquals(1, spans.length);
     Assert.assertEquals(0, spans[0].getStart());
     Assert.assertEquals(2, spans[0].getEnd());
-    Assert.assertEquals(0.8251646041870118, spans[0].getProb(), 0.0);
+    Assert.assertEquals(8.251646041870117, spans[0].getProb(), 0.0);
```

Comment:
Only part that called my attention in the PR. I guess it's just some calculation that changed somewhere, and since it's in the tests, it shouldn't really matter, I think.

Reply:
I think this value is from when I was experimenting with getting it to be a value between 0 and 1 like the other probability values and decided to leave that for later. When I reverted it I didn't also change the value here.